config/database.yml

```diff
@@ -1,25 +1,27 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
   pool: 5
   timeout: 5000
+  username: fabrizio
+  password: <%= ENV['DATABASE_PW'] %>
+  #socket: /var/run/mysqld/mysqld.sock
```

Comment:
@schas002 ok... that is a good idea.. so maybe I keep a `database.yml` where both the sqlite and postgresql parameters are available...but they are just commented out...